### PR TITLE
Equalizer exchange dexs volume

### PR DESCRIPTION
**Pull request by dtmkeng**

### CODE REVIEW

**Comment by llamatester** [comment]
The equalizer-exchange adapter exports: 
        
 
 _______________________________________
    Dexs for 9/1/2023
    _______________________________________
    
    FANTOM 👇
    Backfill start time: 9/12/2022
    NO METHODOLOGY SPECIFIED
    Daily volume: 798964.4669459823
    Timestamp: 1673308798
    
    
    
    
    Done in 16.33s.

**Comment by llamatester** [comment]
The equalizer-exchange adapter exports: 
        
 
 _______________________________________
    Dexs for 11/1/2023
    _______________________________________
    
    FANTOM 👇
    Backfill start time: 9/12/2022
    NO METHODOLOGY SPECIFIED
    Daily volume: 697476.2826527702
    Timestamp: 1673481598
    
    
    
    
    Done in 16.09s.

File: dexs/equalizer-exchange/index.ts [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+        amount1Out,
+      } as IAmount
+    }) as IAmount [];
+    const token0Price = (prices[`fantom:${tokens0[index].toLowerCase()}`]?.price || 0);
+    const token1Price = (prices[`fantom:${tokens1[index].toLowerCase()}`]?.price || 0);
+    const totalAmount0 = log
+      .reduce((a: number, b: IAmount) => Number(b.amount0In)+ Number(b.amount0Out) + a, 0) / 10 ** pairInfos[index].token0.decimals * token0Price;
+    const totalAmount1 = log
+      .reduce((a: number, b: IAmount) => Number(b.amount1In)+ Number(b.amount1Out) + a, 0) / 10 ** pairInfos[index].token1.decimals * token1Price;
+    const untrackAmountUSD = (totalAmount0 + totalAmount1) / 2;
```
Comment (0xngmi) [inline dexs/equalizer-exchange/index.ts]: why count it twice and then divide by 2?

Comment (0xngmi) [reply dexs/equalizer-exchange/index.ts]: would be better if you just counted the token for which we have price data i think?

Comment (dtmkeng) [reply dexs/equalizer-exchange/index.ts]: idk, i follow code base from subgraph uniswap they div by 2
Screenshot 2566-01-13 at 19 24 02

**Comment by llamatester** [comment]
The equalizer-exchange adapter exports: 
        
 
 _______________________________________
    Dexs for 13/1/2023
    _______________________________________
    
    FANTOM 👇
    Backfill start time: 9/12/2022
    NO METHODOLOGY SPECIFIED
    Daily volume: 1720823.9590232112
    Timestamp: 1673654398
    
    
    
    
    Done in 17.44s.

**Comment by llamatester** [comment]
The equalizer-exchange adapter exports: 
        
 
 _______________________________________
    Dexs for 13/1/2023
    _______________________________________
    
    FANTOM 👇
    Backfill start time: 9/12/2022
    NO METHODOLOGY SPECIFIED
    Daily volume: 3441647.9180464223
    Timestamp: 1673654398
    
    
    
    
    Done in 19.80s.

**Comment by dtmkeng** [comment]
@0xngmi I guess it is ok
coingecko
image

**Comment by llamatester** [comment]
The equalizer-exchange adapter exports: 
        
 
 _______________________________________
    Dexs for 19/1/2023
    _______________________________________
    
    FANTOM 👇
    Backfill start time: 9/12/2022
    NO METHODOLOGY SPECIFIED
    Daily volume: 2831981.841167447
    Timestamp: 1674172798
    
    
    
    
    Done in 18.48s.

**Pull request merged**